src/main/java/uk/gov/hmcts/reform/prl/models/dto/ccd/CaseData.java

```diff
@@ -753,17 +753,8 @@ public CaseData setIssueDate() {
     private ResponseDocuments respondentDc8;
     private ResponseDocuments respondentEc8;
 
-    @JsonProperty("respondentAc8Documents")
-    private List<Element<ResponseDocuments>> respondentAc8Documents;
-    @JsonProperty("respondentBc8Documents")
-    private List<Element<ResponseDocuments>> respondentBc8Documents;
-    @JsonProperty("respondentCc8Documents")
-    private List<Element<ResponseDocuments>> respondentCc8Documents;
-    @JsonProperty("respondentDc8Documents")
-    private List<Element<ResponseDocuments>> respondentDc8Documents;
-    @JsonProperty("respondentEc8Documents")
-    private List<Element<ResponseDocuments>> respondentEc8Documents;
```

Comment:
we need to double check on data migration for this. once we remove this fields, any case with this data present will fail. so we must verify